src/sentry/migrations/0662_monitor_drop_last_state_change.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 5.0.2 on 2024-03-05 17:39
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0661_artifactbundleindex_cleanup_step2"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="monitorenvironment",
+            name="last_state_change",
+        ),
```

Comment:
This isn't safe. You'll need to do a state only change first to remove the column from django state so that django isn't querying the column when you drop the column from schema.

Reply:
Specifically, you should follow https://develop.sentry.dev/database-migrations/#deleting-columns

Reply:
👍